### ISSUE TITLE
Manjaro is for the Experienced, not for Beginner

### ISSUE THREAD

**Issue by JiaZhangJiaZhangJiaZhangJiaZhang** [EmeraldSnorlax/manjarno#14]
Perhaps an addition that Manjaro isn't for beginner and their beginner-friendly claim is questionable ...

Manjaro claimed and promoted the distro to be beginner-friendly and community-driven. I tried Manjaro last year December because of this. But, I noticed that the distro wasn't beginner-friendly at all. It's for more experienced users. So their claim feels like marketing claim; I didn't notice before that Manjaro is a company though given their org domain :(

Many youtube videos or websites recommend Manjaro as a beginner-friendly distro other than Ubuntu. But, I believe new GNU/Linux users should be _strongly_ warned that Manjaro isn't for newbie; you need to know Linux kernel, how to interface with tty or use live media chroot when the system is unbootable after an update, etc.

For your latest info that Manjaro really isn't for beginner:
https://forum.manjaro.org/t/kernel-updates-keep-breaking-nvidia-driver/98548
https://forum.manjaro.org/t/black-screen-of-death-after-update-515-rt-nvidia-drivers/101429/52
https://forum.manjaro.org/t/linux-kernel-5-15-18-2-rt28-manjaro-broke-nvidia-drivers/101549

**Comment by EmeraldSnorlax** [comment]
eh - i feel like what's already in the article is enough. [things that you've mentioned above happen across all distros and isnt an issue inherent to manjaro](https://www.youtube.com/watch?v=iYWzMvlj2RQ). i think ive covered most of the manjaro specific ones, but i welcome any specific examples

**Comment by JiaZhangJiaZhangJiaZhangJiaZhang** [comment]
Oh, I think I didn't explain it well. So what I meant was the false claim from the Manjaro company that Manjaro distro was beginner-friendly when it's not.

We can agree with the statement first: Unless you have a *superhuman* QA, a rolling-release distro can barely make it as a beginner-friendly distro; you can get system failure on every update with higher probability than the conventional stable version release distro.

This is especially true if a system uses NVIDIA GPU.

But Manjaro company made such a false claim (given their QA is questionable, you also know this):
"
First paragraph of https://manjaro.org/features/usercases/beginners/
"

Take an example from OpenSUSE, OpenSUSE gives an honest warning about Tumbleweed that it's for the experienced.
"
https://en.opensuse.org/Portal:Tumbleweed
At section **Who should use openSUSE Leap instead of Tumbleweed?** and **Who should try Tumbleweed?**.
"

I switched to Tumbleweed and I've been using it for a while. My impression on Tumbleweed is that Tumbleweed and Manjaro have the **exact** experience in terms of polished look of the DE as well as the wide-range of application availability as claimed by the Manjaro company https://manjaro.org/features/useful-for-everyone/

On why Manjaro company makes such a false claim, this looks more like the company motive to grab users; Manjaro seems more like a company than a community, in contrast with OpenSUSE that is more to a community than a company. Though, I'm just about 50% confident on this, so probably no need to elaborate on this.

The *beginner-friendly* claim along with the wide-range of most recently updated applications provided from the official repo (an inherent nature of rolling-distro) make Manjaro the best distro *magnet* for beginners.

Then the consequence of such false claim is you last conclusion:
```
Maintaining a distro is commendable, and that alone takes credit. However, I'd rather not have
your time (and others, when Manjaro inevitably breaks, and you need to ask for help) wasted,
trying to figure out the odd quirks and issues that Manjaro causes.
```

P.S.

I say the pamac AUR problem is also the consequence of many beginner users; they just simply don't understand AUR and I've seen posts where they mistook AUR with Arch Official Repository.

Off-topic, probably good as more evidences for your part QA stability section:
https://forum.manjaro.org/t/no-audio-in-voice-google-com-calls-since-recent-package-updates/103506
https://forum.manjaro.org/t/freecad-crashes-when-changing-colors/95804
https://forum.manjaro.org/t/gmic-qt-for-krita-5/98472

**Comment by EmeraldSnorlax** [comment]
i still think what is already mentioned is sufficient. this seems to hinge on the idea that manjaro is acting like a company trying to increase user share. while some events (e.g. their treasurer, marketing) might imply that, it is not provable that this is the sole reason why. besides, there are plenty of linux distros backed by companies (e.g. canonical, redhat, suse) so it wouldn't be fair to pin this as a manjaro issue

tldr; manjaro is contradictory and misleading, i agree, but i think the article already gets at in a fair way without holding other distros to certain double standards.
